.gitignore

```diff
@@ -117,7 +119,13 @@ results_processing/*
 docs/formulation/*aux
 docs/formulation/*log
 
+# solver folder for avoiding pushing xpauth.xpr file to remote
+solver/*
```

Comment:
we have to keep this folder tracked in order to run tests via Github actions. please remove it from .gitignore (it is repeated twice).

Reply:
This has been removed via commit 8c71c103233c4c4989dd7bbf8ddd4d44f0449841